services/webnn/webnn_switches.h

```diff
@@ -45,10 +45,18 @@ inline constexpr char kWebNNOrtUseOVModelCache[] =
 // Usage: --no-sandbox --enable-logging --webnn-ort-logging-level=VERBOSE
 // Other severity levels could be "INFO", "WARNING" (default), "ERROR" and
 // "FATAL".
-// Please note if "--use-redist-ort" switch is used, this logging level
-// setting will be ignored, because an OrtEnv will be created before with
-// WARNING logging level.
 inline constexpr char kWebNNOrtLoggingLevel[] = "webnn-ort-logging-level";
+
+// Allow loading onnxruntime.dll from the program files folder firstly. Note, if
+// the switch is not used or fail to load, webnn will try to load
+// onnxruntime.dll from the module folder.
+// Usage: --webnn-ort-load-program-files
```

Comment:
As a second thought, could we allow developers to set ort path with this switch? We can add a comment explaining that for testing within GPU sandbox, developers need to copy ONNX Runtime DLLs, OpenVINO EP DLL and OpenVINO DLLs into a folder under "Program Files" and specify it with this switch, e.g.

```
--webnn-ort-library-path=C:\Program Files\ONNXRuntime-OVEP
```

Reply:
Updated.
Usage: `--webnn-ort-library-path="C:\Program Files\ONNXRuntime-OVEP" --allow-third-party-modules`

PTAL, thanks!